xlconnect.Rmd

```diff
@@ -0,0 +1,544 @@
+#  `XLConnect`: easy and controlled interface to Microsoft Excel
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, collapse = TRUE, eval = FALSE)
+```
+
+MS Excel is probably the most used support to share data analysis and reports in enterprise. Despite its popularity, performing data analysis within an Excel workbook can be cumbersome as well as error prone due to irreproducibility. A possible way out is to perform most of the steps of your data analysis and reporting in another programming language, such as R, and just present and share the results in Excel. 
+Using the package [`XLConnect`](https://github.com/miraisolutions/xlconnect) will allow you to read, write and manipulate MS Excel files from within R.
+
+XLConnect's main features:
+
+- read & write Excel worksheets & named regions
+
+- create, remove, rename, clone & hide / unhide worksheets
+
+- add graphs
+
+- specify / apply cell styles, column width & row height
+
+- define behavior when error cells are encountered
+
+- force formula recalculation when workbooks are opened
+
+- others: merge / unmerge cells, set auto-filters
+
+- etc.
+
+## Installation
+
+XLConnect is cross-platform and runs in all operating systems that support Java. Besides R and a JRE (Java Runtime Environment), nothing else is needed, even an installation of MS Excel is NOT required.
+
+```{r , echo = TRUE, eval = FALSE}
+# Install XLConnect
+install.packages("XLConnect")
```

Comment:
```suggestion
if (!require(XLConnect)) install.packages("XLConnect")
```
it is good practice to skip install.packages in a script if the packages is already installed. not sure if we need to explain the purpose of this if we add it.